server/sonar-server/src/test/java/org/sonar/server/platform/ws/ServerWsTest.java

```diff
@@ -29,7 +31,7 @@
 
 public class ServerWsTest {
 
-  WsTester tester = new WsTester(new ServerWs());
+  WsTester tester = new WsTester(new ServerWs(new ServerWsAction[]{new DummyServerWsAction()}));
```

Comment:
Not sure of the value of testing Pico injection :) instead you could inject a real ServerMigrateWsAction and check that it is correctly declared.